### PR TITLE
Create A 404 page

### PR DESCRIPTION
**Pull request by radioblahaj**

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+   
+   a {
+   border-color:blue;
+   
+</head>
+<body>
+
+<center> 
+<h1> 404 Not Found </h1>
+<p> The Page requested can't be found, If you thing this is wrong <a href="https://github.com/greenbayrules/quark-editor/issues/"> Create A Issue </a>
```
Comment (Sheshank-s) [inline 404.html]: you need to add a closing </p> tag

**Review by Sheshank-s — CHANGES_REQUESTED** [review]

File: 404.html (hunk 1 above)
Comment (radioblahaj) [reply 404.html]: i did

Comment (Sheshank-s) [reply 404.html]: `</p> after the </a>`

Comment (radioblahaj) [reply 404.html]: oh opps

**Comment by Sheshank-s** [comment]
Why remove styles.css?

**Comment by radioblahaj** [comment]
because the style.css would collide with the custom one i made in head

File: 404.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ 
+  <link rel="icon" type="image/jpeg" href="quarklogo.jpg" />
+  
+  <style>
+   h1 {
+   font-family: 'Roboto', sans-serif;
+   
+   a {
+   border-color:blue;
+
```
Comment (Sheshank-s) [inline 404.html:30]: you need to close the `</style>` tag

**Review by Sheshank-s — CHANGES_REQUESTED** [review]

**Comment by callowaysutton** [comment]
Seems good why not ship it?

**Comment by Sheshank-s** [comment]
There are many errors, he didn't close the style tag, etc @callowaysutton

**Comment by Sheshank-s** [comment]
Closing, as I doubt your going to do this @Narulacodes

**Comment by radioblahaj** [comment]
i will! now!

**Comment by radioblahaj** [comment]
Repoen so i can change it!

**Comment by Sheshank-s** [comment]
I'll do it myself

**Comment by MasterOfTheTiger** [comment]
I would love to help, but what's the progress on yours?

**Comment by Sheshank-s** [comment]
@MasterOfTheTiger Nah, it's okay I'm going to do this myself

**Comment by MasterOfTheTiger** [comment]
@GreenBayRules  Okay. Please tell me if there is anything I am able to help with in this repo.

**Comment by Sheshank-s** [comment]
@MasterOfTheTiger Sure thanks! The only thing right now is adding support for Java and Python and I think @callowaysutton is on to that :D Thanks for the help!

**Pull request closed without merging**